app/com/yetu/oauth2provider/views/confirmmail.scala.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+			<title>Confirm Mail</title>
+			<link rel="stylesheet" media="screen" href="@controllers.routes.Assets.at("stylesheets/setup.css")">
+			<link rel="stylesheet" media="screen" href="@controllers.routes.Assets.at("stylesheets/confirmmail.css")">
+			<link rel="shortcut icon" type="image/png" href="">
+			<meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+	</head>
+	<body>
+		<header>
+			<h1>@Messages("setup.headline")</h1>
+			<h2>@Messages("setup.subheadline")</h2>
+		</header>
+		<div class="steps">
+			<ul class="steplist">
+				<li class="step active">@Messages("setup.steps.one")</li>
+				<li class="step active">@Messages("setup.steps.two")</li>
+				<li class="step">@Messages("setup.steps.three")</li>
```

Comment:
much of this code is an exact copy of `confirmedmail.scala.html`

Make use of the twirl templating and include parts into the other html. See https://playframework.com/documentation/2.3.x/ScalaTemplates e.g. reusable blocks or also see the example of including one part into another from the main very first play framework example, here;
https://github.com/typesafehub/activator-hello-play-scala/tree/master/app/views